docs/data-sources/external_links.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: External Links"
+description: |-
+    Get the information about all Wavefront External Links.
+---
+
+# Data Source: wavefront_external_links
+
+Use this data source to get information about all Wavefront external links.
+
+## Argument Reference
+* `limit` - (Optional) Limit is the maximum number of results to be returned. Defaults to 100.
+* `offset` - (Optional) Offset is the offset from the first result to be returned. Defaults to 0.
+
+## Example Usage
+
+```hcl
+# Get the information about all external links.
+data "wavefront_external_links" "example" {
+  limit = 10
+  offset = 0
+}
+```
+
+## Attribute Reference
+
+* `external_links` - List of all external links in Wavefront. For each external link you will see a list of attributes.
+  * `name` - The name of the external link.
+  * `id` -  The ID of the external link.
+  * `description` - Human-readable description for this link.
+  * `template` - The mustache template for this link. The template must expand to a full URL, including scheme, origin, etc.
```

Comment:
The mustache template for the link. The template must expand to a full URL, including scheme, origin, etc.